data/text/obtain_item.inc

```diff
@@ -29,3 +29,9 @@ gText_NoRoomLeftForAnother::
 gText_TheDecorWasTransferredToThePC::
 	.string "The {STR_VAR_2} was transferred\n"
 	.string "to the PC.$"
+
+gText_ObtainedTheItems::
+    .string "Obtained {STR_VAR_1} {STR_VAR_2}!$"
+
+gText_PlayerFoundItems::
+    .string "{PLAYER} found {STR_VAR_1} {STR_VAR_2}!$"
```

Comment:
Tabs instead of spaces

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/3942/commits/9350a93cdf6f950be97a2665798693fa05946cd6